mod-quick-marc/src/main/resources/spitfire/mod-quick-marc/features/setup/samples/setup-records/authority-source-file.json

```diff
@@ -1,14 +1,12 @@
 {
   "id": "af045f2f-e851-4613-984c-4bc13430454a",
   "name": "LC Name Authority file (LCNAF)",
-  "codes": [
-    "n",
-    "nb",
-    "nr",
-    "no",
-    "ns"
-  ],
+  "code": "abcdefghijklmnopqrstuvwxy",
   "type": "Names",
   "baseUrl": "id.loc.gov/authorities/names/",
-  "source": "folio"
+  "source": "folio",
```

Comment:
new request schema for authority source file creation (POST endpoint) does not have ```source``` field now, please remove source from here